floodwatch/src/js/components/Profile.js

```diff
@@ -0,0 +1,198 @@
+// @flow
+
+import React, { Component } from 'react';
+import { Row, Col, Button, ListGroup, ListGroupItem } from 'react-bootstrap';
+import _ from 'lodash'
+import Filters from '../../stubbed_data/filter_response.json';
+import DemographicKeys from '../../stubbed_data/demographic_keys.json';
+import type {DemographicDictionary} from './FindInDemographics'
+
+import {FWApiClient} from '../api/api';
+import type {PersonDemographicRequest} from '../api/types';
+
+type StateType = {
+  userData?: PersonDemographicRequest,
+  toPick: Array<string>,
+  curStatus: string
```

Comment:
This is a good place to use the "enum" type that comes with FlowType. You could annotate this as `curStatus: null | "success" | "error"` and then be guaranteed you don't set it to anything else later.